server/methods/accounts/accounts.js

```diff
@@ -580,7 +580,7 @@ export function inviteShopOwner(options) {
 
   Reaction.Email.send({
     to: email,
-    from: `${shop.name} <${_.get(shop, "emails[0].address")}>`,
+    from: `${_.get(dataForEmail, "primaryShop.name")} <${_.get(dataForEmail, "primaryShop.emails[0].address")}>`,
```

Comment:
@spencern I have no reason for using `_.get` vs. just `dataFormEmail.primaryShop.name` here, I just followed the convention we were currently using. If it's better not to invoke `lodash`, I will change it.

Reply:
I'd prefer not to use lodash here, I don't think `_.get` is very clear unless you're recently versed in lodash terminology, so if you don't mind gutting it, that'd be better.

Reply:
reverted back to using _.get() as discussed

Reply:
For anyone following along at home, decided to keep `_.get` for now because it automatically guards against throwing an TypeError when trying to access a property of something that is undefined.

We're currently using `_.get` throughout the app like this and we should discuss what lodash features are permitted within Reaction and either standardize on using _.get, or eliminate it.